From 80ad9df4ddaf40b14c2fe380c57efe25e6d92226 Mon Sep 17 00:00:00 2001
From: Varsha Maira <vmaira@habana.ai>
Date: Mon, 09 Jun 2025 21:40:18 +0300
Subject: [PATCH] Revert "[SW-230471] g3: mgmt: disable assert to reduce iccm usage"

This reverts commit ebcdff48c8efbcb0044e0ef39bc3c3d41f0bb9fe.

Keeping the assert for 1.22 release

Change-Id: Iec8086ec9576463005b5e86bfeb9ee820c9a51c2
---

diff --git a/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig b/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
index 30b250b..6e9e6bb 100644
--- a/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
+++ b/boards/hl/gaudi3/hl_gaudi3_mgmt_asic_defconfig
@@ -162,6 +162,6 @@
 CONFIG_LFS_FOLDERS_SUPPORT=n
 CONFIG_LFS_LOG_FILE_SIZE=524288
 CONFIG_LFS_FORMAT_IF_MOUNT_FAILED=n
-CONFIG_ASSERT_VERBOSE=n
+CONFIG_ASSERT_VERBOSE=y
 CONFIG_PINCTRL=y
 CONFIG_PINCTRL_HL_GAUDI3=y
